cost-analyzer/templates/forecasting-service.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.forecasting.enabled }}
+
+kind: Service
+apiVersion: v1
+metadata:
+  name: {{ template "forecasting.serviceName" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "forecasting.commonLabels" . | nindent 4 }}
```

Comment:
```suggestion
    {{ include "forecasting.commonLabels" . | nindent 4 }}
```

Reply:
This is correctly indented when I open the file in my local editor.

Reply:
Juust kidding, different file, will fix